### ISSUE TITLE
Driver ErrorNo method causes process kill

### ISSUE THREAD

**Issue by 3b-fly** [ADOdb/ADOdb#254]
If mysqli type is used and mysqli extension is not loaded, connection ErrorNo method is called but it is overriden by mysqli ErrorNo method which calls mysqli extension functions. It ends with process kill with no exception.

**Comment by dregad** [comment]
Does this occur when using adodb-exceptions ?

**Comment by dregad** [comment]
I managed to reproduce the problem using exceptions

``` php
<?php
include('adodb.inc.php');
include('adodb-exceptions.inc.php');
$db = ADONewConnection('mysqli');
$db->connect('localhost', 'username', 'password', 'database') or die('ARGH');
```
PHP dies with fatal error.

**Comment by 3b-fly** [comment]
It does die without any error even with adodb-exceptions included in v5.20.4 - PHP 5.2.4.
